content/docs/start/experiments-trail/experiments-1.md

```diff
@@ -0,0 +1,334 @@
+```
+
+### Get the data set
+
+The repository you cloned doesn't contain the dataset. In order to get
+`fashion-mnist.tar.gz` from `dataset-registry`, we use `dvc get` to download the
+missing data files. `dvc get` is similar to `wget` but works with Git+DVC
+repositories to download binary files.
+
+```dvc
+$ dvc get https://github.com/iterative/dataset-registry \
+          fashion-mnist/images.tar.gz -o data/images.tar.gz
+```
+
+Then we extract this file that contains labeled images.
+
+```dvc
+$ tar -xvzf data/images.tar.gz --directory data/
+```
+
+### Specify the experiment
+
+We first initialize DVC inside the project to create an experiment.
+
+```dvc
+$ dvc init
+```
+
+Then we add the dataset to the project:
+
+```dvc
+$ dvc add data/images
+```
+
+This creates a `data/images.dvc` file that contains all the relevant metadata of
+the directory. You can add and commit `data/images.dvc` file, and changes in
+`.gitignore` that hides the `data/images/` directory from Git.
+
+```dvc
+$ git add data/.gitignore data/images.dvc
+$ git commit -m "Dataset added"
+```
+
+DVC experiments are run by specifying their commands, outputs, parameters and
+dependencies. We specify an experiment command by `dvc stage add`.
+
+```dvc
+$ dvc stage add -n train \
+                -p model.conv_units \
+                -p train.epochs \
+                -d data/images \
+                -m metrics.json \
+                python3 src/train.py
+```
+
+The command tells DVC to create an experiment named `train`, and that for any
+change in `data/images/`, `model.conv_units` or `train.epochs`, we run an
+experiment using `src/train.py` that produces a new `metrics.json` file.
+
+DVC creates `dvc.yaml` file and modifies `.gitignore` for Git to ignore certain
+artifacts.
+
+```dvc
+$ git add dvc.yaml .gitignore
+$ git commit -m "added data and the experiment"
+```
+
+DVC is ready to run the experiments now!
+
+</details>
+
+## Running the experiment with default parameters
+
+The purpose of `dvc exp` subcommands is to run the pipeline for ephemeral
+experiments. By _ephemeral_ we mean the experiments can be run without
+committing parameter and dependency changes to Git. Instead the artifacts
+produced for each experiment are tracked by DVC and persisted on demand.
+
+Running the experiment with default hyperparameter values requires only the
+command:
+
+```dvc
+$ dvc exp run
+...
+Reproduced experiment(s): exp-7683f
+Experiment results have been applied to your workspace.
+
+To promote an experiment to a Git branch run:
+
+        dvc exp branch <exp>
+```
+
+It runs the pipeline starting from the basic dependencies and produces
+`metrics.json` file for the default state.
+
+<details>
+
+### If you used `dvc repro` before
+
+Earlier versions of DVC uses `dvc repro` to run the pipeline. If you already
+have a DVC project, you may already be using `dvc repro`.
+
+In DVC 2.0 `dvc exp run` supersedes `dvc repro`. Both of these commands run the
```

Comment:
I don't think `exp run` is supposed to replace `repro`. Esp not in the DVC 3.0 directions AFAIK.
```suggestion
have a DVC project, you may already be using `dvc repro`. Both of these commands run the
```

Reply:
`dvc exp run` does the same thing as `dvc repro`, namely _running the pipeline._ A user that knows _dvc repro_ will recognize that this new command does the same thing as `dvc repro`. In 3.0, there may be _experiments without pipeline_, but that's another topic and even in that case, `dvc exp run` will continue to _run the pipeline_ in `dvc.yaml`.

Reply:
> dvc exp run does the same thing as dvc repro

It's not 100% the same but again, that may change in the near future so I wouldn't see it this way. I don't think it was really the intention. The intention is that repro is the interface to eval and exec pipelines (dvc.yaml) while `exp run` puts project versions (executes with `repro` internally if needed) into virtual workspaces that DVC tracks automatically.

Maybe there's no need to even mention `repro` at all TBH. @shcheklein can probably confirm here.

Reply:
I guess my point is that this whole section emphasizes that `exp run` is the new `repro` and that's not the case. If the idea is to have a mini migration guide for existing DVC repos, a `<details>` section may suffice.